community/github-cli/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+make 
+
+./bin/gh completion -s bash > bash
+./bin/gh completion -s zsh > zsh
```

Comment:
Do we really need to include these? To silence shellcheck just change `> file` to `> ./file`

Reply:
Yes and No. If you like the convenience of completion then yes if you can live without it then no, I assumed the former as it is something you can negligibly live with.